client/app/services/group/currentGroup.service.js

```diff
@@ -14,7 +14,18 @@ export default class CurrentGroup {
     Object.assign(this, {
       value: {},
       User,
-      SessionUser
+      SessionUser,
+      map: {
```

Comment:
Yes, they are. I thought it's the best place to store them, since all of the map is group related and I didn't want to create another "currentGroupMap" service. Open for ideas though.